ideascube/conf/kb_rwa_giz.py

```diff
@@ -0,0 +1,13 @@
+"""KoomBook conf"""
+from .kb import *  # noqa
+
+LANGUAGE_CODE = 'en'
+IDEASCUBE_NAME = 'GIZ'
```

Comment:
> I'm using set server name currently in the playbook, however the name picked up to set the var is the project name which is not really nice...

Well, but that could be made nicer with a separate variable, set to a prettier name, no?